redis/src/test/scala/zio/redis/ScriptingSpec.scala

```diff
@@ -0,0 +1,279 @@
+package zio.redis
+
+import zio._
+import zio.redis.RedisError._
+import zio.test.Assertion._
+import zio.test.TestAspect.ignore
+import zio.test._
+
+trait ScriptingSpec extends BaseSpec {
+  val scriptingSpec: Spec[Annotations with RedisExecutor, TestFailure[Any], TestSuccess] =
```

Comment:
Lots of tests seems to be ignored, what's the problem?

Reply:
test for flash command need specific test environment to run, due to constancy i wrote ignore.
with other tests i was straggling to wrote them that they would work like i want. For example test for command kill i don;t understand why but don't want to kill the script((, test for debug when you start command yes or sync block evrething in redis and don't want to turn off debug mode. 
I think that debug, kill should be run to in specific environment.

Reply:
Hm, in that case i propose shipping the testable commands in this PR and the other through follow up pull requests, i.e. once the env is figured out.

Reply:
Ok